lib/hammer_cli_foreman/user.rb

```diff
@@ -90,6 +90,9 @@ class DeleteCommand < HammerCLIForeman::DeleteCommand
     lazy_subcommand('mail-notification', _("Managing personal mail notifications"),
                     'HammerCLIForeman::UserMailNotification', 'hammer_cli_foreman/user_mail_notification'
     )
+    lazy_subcommand('table-preferences', _("Managing table preferences"),
+                    'HammerCLIForeman::TablePreferences', 'hammer_cli_foreman/table_preferences'
```

Comment:
Let's stick with our conventions:
```suggestion
    lazy_subcommand('table-preference', _("Managing table preferences"),
                    'HammerCLIForeman::TablePreference', 'hammer_cli_foreman/table_preference'
```